justfile

```diff
@@ -41,11 +41,11 @@ test-pattern-default := "/"
 # The endpoints tested may be limited by supplying a test pattern in the form "/endpoint_1|enpoint_2|..|enpoint_n"
 # For example, to run the testing suites for eth_chainId & eth_blockNumber you should run:
 # `just run-hive "/eth_chainId|eth_blockNumber"`
-run-hive test-pattern=test-pattern-default: build_image
-    cd hive && ./hive --sim "ethereum/rpc-compat" --client ethereumrust --sim.limit "{{test-pattern}}" 
+run-hive simulation test-pattern=test-pattern-default: build_image setup-hive
+    cd hive && ./hive --sim {{simulation}} --client ethereumrust --sim.limit "{{test-pattern}}" --docker.output
```

Comment:
Please remove the `--docker.output` flag

Reply:
good catch